app/src/main/java/edu/uc/haflga/financialpal/dto/Limit.kt

```diff
@@ -1,4 +1,4 @@
 package edu.uc.haflga.financialpal.dto
 
-// THIS IS THE LIMIT CLASS FOR THE LIMIT AMOUNT FOR DIFFERENT TRANSACTION CATEGORIES
+/** THIS IS THE LIMIT CLASS FOR THE LIMIT AMOUNT FOR DIFFERENT TRANSACTION CATEGORIES */
```

Comment:
Changed this comment to the proper javadoc format for a one line comment

Reply:
Spread this out to multiple lines:
/**
 *
 */